app/src/main/java/com/example/weather/CityPickerViewModel.kt

```diff
@@ -3,35 +3,38 @@ package com.example.weather
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.viewModelScope
 import com.example.weather.network.CityOverview
-import com.example.weather.network.WeatherApi
-import com.example.weather.network.WeatherApiResponse
+import com.example.weather.repository.CityRepository
 import kotlinx.coroutines.launch
 
 const val cityIds = "4772354,5368361,5128581,4887398,4699066,5809844,5856195,5419384,5391959,4684888,5746545,4990729,4335045"
 const val appId = "bd4472d97ca0b479dc32513cf50a1bf3"
 const val units = "metric"
 
-class CityPickerViewModel : ViewModel() {
+class CityPickerViewModel(private val repository: CityRepository) : ViewModel() {
 
-    private var _cities = MutableLiveData<List<CityOverview>>()
-
-    var cities: LiveData<List<CityOverview>> = _cities
+    private var _cities = repository.cityOverview
+    var cities: MutableLiveData<List<CityOverview>> = _cities
```

Comment:
this should be a `val` instead of a `var` and should be of type `LiveData` instead of `MutableLiveData`. When you do these backing variables, you want the publicly exposed one to be a val and immutable. That way, other classes are only able to read this data and not update it or modify it in any way as intended